spec/controllers/whitehall_media_controller_spec.rb

```diff
@@ -52,7 +50,7 @@
       it 'redirects to thumbnail-placeholder image' do
         get :download, path: path, format: format
 
-        expect(controller).to redirect_to(image_path('thumbnail-placeholder.png'))
+        expect(controller).to redirect_to(ActionController::Base.helpers.image_path('thumbnail-placeholder.png'))
```

Comment:
Similarly, I think this could be written more simply as:

```ruby
expect(controller).to redirect_to(controller.class.helpers.image_path('thumbnail-placeholder.png'))
```

Or possibly:

```ruby
expect(controller).to redirect_to(described_class.helpers.image_path('thumbnail-placeholder.png'))
```

Reply:
We've updated this to use `described_class.helpers` as you suggest.